shell/utils/pagination-utils.ts

```diff
@@ -95,15 +95,7 @@ class PaginationUtils {
         return setting === enabledFor.resource?.id;
       }
 
-      if (setting.resource === enabledFor.resource?.id) {
-        if (!!setting.context) {
-          return enabledFor.resource?.context ? setting.context.includes(enabledFor.resource.context) : false;
```

Comment:
It's not obvious to me why this is necessary? It seems to cause problems if you want to use the PaginatedResourceTable on non-list pages.

Reply:
This whole block needs to stay.

`if (setting.resource === enabledFor.resource?.id) {` - used to ensure the correct entry is found
`if (!!setting.context) {` - the context was used to explictely block pagination outside of supported areas. For instance the prov cluster was only allowed in the side nav and home page and not the cluster management list